internal/pkg/term/selector/selector.go

```diff
@@ -960,6 +954,31 @@ func (s *ConfigSelector) Job(msg, help, app string) (string, error) {
 	return selectedJobName, nil
 }
 
+func (s *ConfigSelector) Workload(msg, help, app string) (string, error) {
```

Comment:
Can we put comment to this public function? 
And where are we calling this function from?

Reply:
> Can we put comment to this public function?

done!

> And where are we calling this function from?

we will need it in `storage init`